src/js/actions/tools.js

```diff
@@ -791,6 +792,39 @@ define(function (require, exports) {
             fn: _vectorMaskHandler
         });
 
+        _vectorMaskPolicyHandler = function () {
+            var vectorMaskMode = toolStore.getVectorMode() || false;
+
+            if (vectorMaskMode) {
+                var pointerPolicyID = toolStore.getVectorMaskPolicyID();
+         
+                if (pointerPolicyID) {
+                    return this.transfer(policy.removePointerPolicies, pointerPolicyID)
+                        .bind(this)
+                        .then(function () {
+                            var cloakRect = uiStore.getCloakRect(),
```

Comment:
Fix the tabbing here. Also, can you factor this out into a private helper function that is usable both here and above? Something like `_getVectorMaskPolicyBounds`?